sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -218,6 +220,10 @@ export class ConfirmationOfEnrollmentController {
         "Confirmation of Enrollment and application status update to completed is failed",
       );
     }
+
+    await this.disbursementScheduleService.updateDisbursementScheduleDocumentNumber(
```

Comment:
It is nice to have the number being generated in a separated function, but should it not be saved altogether with the status change of the COE on the 212? What happens if the COE is updated but the execution of the line 224 fails?
I believe that it should be all together in the same "transaction" to avoid DB inconsistences.